Servers/driver/autoDriver.driver.ts

```
@@ -485,39 +461,268 @@ const insertQuery: TableList = [
+    generateValuesString: userGenerateValuesString,
+  } = insertQuery["users"];
+  let users;
+  if (userMockData.length !== 0) {
+    const values = userMockData(-1, roles![0].id, roles![1].id, roles![2].id).map((d) => userGenerateValuesString(d as any));
+    insertString += values.join(",") + "RETURNING id;";
+    users = await insertData(insertString as string);
+  }
+
+  var {
+    mockData: projectMockData,
+    tableName,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'tableName'. Consider to delete it or rename it.

'tableName' is defined here:


(lint/suspicious/noRedeclare)